### ISSUE TITLE
[Turing Data Story] Baseball Statistics

### ISSUE THREAD

**Issue by edaub** [alan-turing-institute/TuringDataStories#122]
# Story description 

Given my love of baseball and statistics (my dog-eared copy of *Moneyball* is still a favorite to re-read from time to time) I am interested in writing a story using baseball statistics. Still thinking about the exact details, but it will use the [Lahman Database](http://www.seanlahman.com/baseball-archive/statistics/), so for the moment this is a placeholder to record my interest.

# Ethical guideline 

Ideally a Turing Data Story has these properties and follows the [5 safes framework](http://blog.ukdataservice.ac.uk/access-to-sensitive-data-for-research-the-5-safes/).

- [x] The analysis you produce is openly available and reproducible.
- [x] Any data used are [open](https://opendatahandbook.org/guide/en/what-is-open-data/) and have an explicit licence, provenance and attribution.
- [ ] Any data used are not [personal data](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/key-definitions/what-is-personal-data/) (i.e. the data is anonymous or anonymised).
- [x] Any linkage of datasets in your data story does not lead to an increased risk of the personal identification of individuals.
- [x] The Story must be truthful and clear about any limitations of analysis (and potential biases in data).
- [x] The Story will not lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice.

# Current status

 - [x] Scoping of the story
 - [x] Write story outline
 - [x] Add material to the story notebook
 - [x] Combine materials into a readable story
 - [x] Proofread
 - [x] Request reviews
 - [ ] Address reviews
 - [ ] Merge to master branch.

**Comment by crangelsmith** [comment]
This sounds amazing @edaub 🤩. Please go ahead when you have the time and let us know if we can help in any way.

**Comment by edaub** [comment]
Update following today's discussion with @jack89roberts at the REG Tech Talk:

I spent the afternoon digging more into the SQL database holding the stats. I extracted stats for several years, converted into a more useful form (baseball has this strange definition of an at-bat where they don't count all of the times you come up depending on if certain outcomes happen, which is decidedly *not* the correct way to handle it), and then fit some Bayesian models based on a multinomial distribution to estimate the league average. The idea is to do some kind of Bayesian analysis to measure marginal utility (also known as "replacement level" in baseball parlance) with some kind of uncertainty bounds, and compare players to replacement level/compare replacement levels across different eras. Fortunately at least in simple cases this can be done analytically, so hopefully I'll be able to avoid needing to do any heavy computation in this analysis.

This is fairly standard stuff for baseball stat heads, but there doesn't seem to be much usage of Bayesian inference in the popular (or even academic) literature so I think that this is somewhat of a fairly novel approach.
